### ISSUE TITLE
Add icon properties to LabeledBtn component

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#173]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would be to add two properties and their implementations to the [frontend/components/LabeledBtn.vue](https://github.com/activist-org/activist/blob/main/frontend/components/LabeledBtn.vue) component. We want both properties to be optional, with the first being `leftIcon` and the second being `rightIcon`. Each takes a string and would thus allow for identifies of icons to be passed to `<Icon>` components in the `name` property.

An example of a `LabeledBtn` with `leftIcon` as `bi:pencil-square` would be:

![Mode=Dark, Mobile=True, Icon=Edit, Hover=False](https://github.com/activist-org/activist/assets/24387426/2efbf1df-e438-4b33-b91d-5845bc492f3e)

### Contribution

This would be a great first issue for someone who wants some experience with Vue/Nuxt/Tailwind, but is of course open to current contributors :) Happy to discuss, support, or work on it myself if no one has interest 😊

**Comment by tarunsamanta2k20** [comment]
I would like to work on it.

**Comment by andrewtavis** [comment]
Thank you, @tarunsamanta2k20! Will assign you and review in the coming days :)

**Comment by andrewtavis** [comment]
Thanks for your interest in activist 😊

**Comment by andrewtavis** [comment]
@tarunsamanta2k20, I'm going to add some icons to the buttons that we already have in the UI 😊 Once those are added I'll close this issue 🚀

**Comment by andrewtavis** [comment]
@tarunsamanta2k20, eeacf6e closes this. I needed to change how we were writing the component in the call so that the `<Icon>` components were actually within the `<a>` and `<NuxtLink>` tags rather than above and below. I also added a similar `<div>` tag so that we can now use the button for actions that aren't links such as getting a popup 😊 Thanks for your work on this! Really is something great for activist to have this up and running :)
